bedrock/mozorg/templates/mozorg/projects/mozilla-based.html

```diff
@@ -28,7 +28,7 @@ <h3><a href="http://www.seamonkey-project.org/"><img src="{{ static('img/project
   <p>{{ _('The all-in-one application formerly known as the “Mozilla Application Suite”, containing a web browser, a mail and newsgroups client, an HTML editor, web development tools, and an IRC chat client.') }}</p>
   </li>
   <li>
-  <h3><a href="/projects/calendar/"><img src="{{ static('img/projects/calendar/calendar-logo.png') }}" width="64" height="64" alt="" /><span>{{ _('Lightning') }}</span></a></h3>
+  <h3><a href="https://www.thunderbird.net/calendar/"><img src="{{ static('img/projects/calendar/calendar-logo.png') }}" width="64" height="64" alt="" /><span>{{ _('Lightning') }}</span></a></h3>
```

Comment:
Just noting that this URL is still 404.

Reply:
It looks like the logo here `projects/calendar/calendar-logo.png` is still being used, but it's removed in this PR below.

Reply:
I've moved it into the mozilla-based media assets folder and updated the reference.

Thank you for the very through review!